expertise/service/expertise.py

```diff
@@ -70,7 +75,10 @@ def _prepare_config(self, request) -> dict:
         # Create directory and config file
         if not os.path.isdir(config.dataset['directory']):
             os.makedirs(config.dataset['directory'])
-        config.save()
+        with open(os.path.join(config.job_dir, 'config.json'), 'w+') as f:
+            json.dump(config.to_json(), f, ensure_ascii=False, indent=4)
+        self.logger.info(f"Saving processed config to {os.path.join(config.job_dir, 'config.json')}")
+        config.save(self.redis_args)
```

Comment:
why are you saving `config.json` in both memory and redis?

Reply:
I think we should still save some record of the job config on the disk and not just in Redis, for administrative purposes. The big difference is that this file is written once and never interacted with again by the API